### DISCUSSION TITLE
Allow redirects to http urls using other testing TLDs

### DISCUSSION THREAD

**Discussion by apexskier** [ory/hydra#3294] [Ideas]
Related to https://github.com/ory/hydra/issues/1021

Right now, hydra requires explicitly enabling http urls using the `dangerous-allow-insecure-redirect-urls` CLI parameter unless using a `.localhost` domain.

Our local setup chose to use `.test` over `.localhost` a long time ago, so this causes a bit of friction.

[Given `.localhost` comes from RFC 2606](https://datatracker.ietf.org/doc/html/rfc2606), I'd like to propose adding `.test` as an enabled-by-default http callback URL, since it's also a reserved TLD used for testing.

I don't think it'd make sense to do this for `.invalid` or `.example` since they aren't intended for running code.

**Comment by vinckr** [comment]
Hey Cameron, 
thanks for sharing! 
I would like to gauge how many in the community would be interested in this. In general I think most people use .localhost, but that could be wrong.
